### DISCUSSION TITLE
Undefined offset: 0 in phpoffice\phpspreadsheet\src\PhpSpreadsheet\Reader\Xls.php on line 4501

### DISCUSSION THREAD

**Discussion by Lydiachl** [PHPOffice/PhpSpreadsheet#2457] [Q&A]
There is a notice in my program, but it seems didn't effect anything.
However I still want to know how to solve this error.

**Comment by oleibman** [comment]
If you can upload the file that is causing this problem, I can take a look.

**Reply by Lydiachl** [reply]
[test.zip](https://github.com/PHPOffice/PhpSpreadsheet/files/7759206/test.zip)
Hi, I've upload the file and hide some sensitive data (account and password).
This program is used to retrieve file from ftp and generate new file from ftp file data.
It is very kind for you taking a look on it.

**Reply by oleibman** [reply]
Sorry, you uploaded the program which reads the file with the problem, but not the file itself. I can't really investigate without that.

**Reply by Lydiachl** [reply]
Thank you.
